src/vdaf/poplar1.rs

```diff
@@ -128,6 +130,24 @@ pub struct Poplar1InputShare<const SEED_SIZE: usize> {
     corr_leaf: [Field255; 2],
 }
 
+impl<const SEED_SIZE: usize> ConstantTimeEq for Poplar1InputShare<SEED_SIZE> {
+    fn ct_eq(&self, other: &Self) -> Choice {
+        // We short-circuit on the length of corr_inner being different. Only the content is
+        // protected.
+        if self.corr_inner.len() != other.corr_inner.len() {
+            return Choice::from(0);
+        }
+
+        let mut rslt = self.idpf_key.ct_eq(&other.idpf_key)
```

Comment:
nit: I think elsewhere we're using `res` or `result` for a generic result. (I haven't checked.)
```suggestion
        let mut res = self.idpf_key.ct_eq(&other.idpf_key)
```